src/components/Date/Date.jsx

```diff
@@ -84,18 +84,18 @@ class Date extends React.Component {
     }
 
     return (
-      <div className={!isValid ? 'input-error-date' : undefined}>
-        <label>
-          {this.props.label ? this.props.label : 'Date of birth'}
+      <fieldset className={!isValid ? 'input-error-date' : undefined}>
+        <legend className="vads-u-font-size--base vads-u-font-weight--normal">
+          {this.props.label || 'Date of birth'}
           {this.props.required && (
             <span className="form-required-span">(*Required)</span>
           )}
-        </label>
+        </legend>
         {errorSpan}
         <div
           className={isValid ? undefined : 'usa-input-error form-error-date'}
         >
-          <div className="usa-date-of-birth">
+          <div className="usa-date-of-birth usa-datefields clearfix-text">
```

Comment:
Not a fan of keeping `usa-date-of-birth` but there are some definitions that need it.

Also added the `clearfix-text` to override the floats on the date fields. I was going to add a flex box class here, but it's out of scope.

Reply:
What exactly are these new classes doing for us?

Reply:
I added `usa-datefields` as a placeholder for the wrapper, so that eventually we can remove `usa-date-of-birth` and use this new class name. Right now, it's being targeted by [two](https://github.com/department-of-veterans-affairs/vets-website/blob/mater/src/applications/disability-benefits/all-claims/sass/disability-benefits.scss#L44) [different](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/financial-status-report/sass/financial-status-report.scss#L115) forms to adjust margin & padding. It's also needed by [HLR's analytics](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/wizard/pages/bdd.jsx#L95) because a classname is needed for as a field type (for the wizard only).

The `clearfix-text` was added because the month & day selects and year input are all aligned using float. If you look at the screenshots in #15679 you'll see that the float isn't being cleared properly. I really wanted to change this to use flexbox, but I think that's out of scope for this PR.

Reply:
Oh, I see. So `usa-datefields` doesn't add styling, it is _just_ there to be a target?